tdrs-backend/tdpservice/search_indexes/parsers/test/test_category2.py

```diff
@@ -0,0 +1,47 @@
+"""Unit tests for category2.py."""
+
+from tdpservice.search_indexes.models import T1
+from tdpservice.search_indexes.parsers.validators import category2
+from tdpservice.search_indexes.parsers.validators.category2 import validate_cat2, validate, create_document
+
+
+def test_validate_cat2(mocker):
+    """Test validate_cat2."""
+    mocker.patch(
+        'tdpservice.search_indexes.parsers.validators.category2.create_document',
+        return_value=())
+    mocker.patch(
+        'tdpservice.search_indexes.parsers.validators.category2.validate',
+        return_value=())
+
+    spy_create_document = mocker.spy(category2, 'create_document')
+    condition = {'in': [1, 2]}
+    name = 'TEST'
+    value = 'foo'
+    model_obj = T1()
+    validate_cat2(name, value, condition, model_obj)
+    assert spy_create_document.call_count == 0
+
+    condition = {'FOO': {'gt': 0}}
+    name = 'FOO'
+    value = 'bar'
+    model_obj = T1()
+    validate_cat2(name, value, condition, model_obj)
+    assert spy_create_document.call_count == 1
```

Comment:
This is meant to be the pass case.  I had trouble directly testing the else because there wasn't much in it I could hook into.